### PR TITLE
PRO-3936: Update to pass through signature to grant documents.

### PR DESCRIPTION
**Pull request by davejones74**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/PRO-3936

### Change description ###
New signature appears on issued grants
Intestacy - Letter of administration
Digital Grant - Grant of Probate
Admon Will - Letter of administration with will
Will Lodgement Receipt Deposit

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by davejones74** [comment]
Closing PR to add the missing encryption routines for signature image (which I thought was part of another ticket). Will reopen PR once this is once.

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -79,7 +80,7 @@ spring.jackson.serialization.write_dates_as_timestamps: false
 
 ####Evidence management Service URL's
 #hard coded to point to docker container ccd-data-store-api CCD_DM_DOMAIN value
-evidence.management.host: http://localhost:8080
+evidence.management.host: http://dm-store:8080
```
Comment (RebeccaBaker) [inline src/main/resources/application.yml]: Please revert back to localhost not dm-store

**Review by RebeccaBaker — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/probate/service/template/pdf/PDFManagementService.java [hunk 2]
```diff
@@ -39,11 +51,15 @@
     private final ObjectMapper objectMapper;
     private final HttpServletRequest httpServletRequest;
     private final PDFServiceConfiguration pdfServiceConfiguration;
+    private final FileSystemResourceService fileSystemResourceService;
+    
+    static final String SIGNATURE_DECRYPTION_IV = "P3oba73En3yp7ion";
```
Comment (sanjayparekh-hmcts) [inline src/main/java/uk/gov/hmcts/probate/service/template/pdf/PDFManagementService.java:56]: we may want to store this on vault too

Comment (davejones74) [reply src/main/java/uk/gov/hmcts/probate/service/template/pdf/PDFManagementService.java:56]: The Initialisation Vector doesn't need to be a secret value (I.e. stored in the Vault). From what I've read it's best to change the IV for each encryption which I'm documenting in confluence but that relies on updating the encrypted image each time with the new IV which isn't that practical in our use case.

From Wikipedia...
An initialization vector has different security requirements than a key, so the IV usually does not need to be secret.

**Pull request merged**